spec/models/manageiq/providers/openstack/cloud_manager/refresher_spec.rb

```diff
@@ -47,61 +53,102 @@
-  def assert_specific_auth_key_pair
-    key = @ems.key_pairs.find_by(:name => "test-key")
-    expect(key.name).to eq("test-key")
-    expect(key.type).to eq("ManageIQ::Providers::Openstack::CloudManager::AuthKeyPair")
-  end
+    def assert_specific_auth_key_pair
+      key = @ems.key_pairs.find_by(:name => "manageiq-spec-key")
+      expect(key.name).to eq("manageiq-spec-key")
+      expect(key.type).to eq("ManageIQ::Providers::Openstack::CloudManager::AuthKeyPair")
+    end
 
-  def assert_specific_cloud_volume
-    vol = @ems.cloud_volumes.find_by(:name => "test-vol")
-    expect(vol.name).to eq("test-vol")
-    expect(vol.status).to eq("available")
-    expect(vol.volume_type).to eq("__DEFAULT__")
-    expect(vol.type).to eq("ManageIQ::Providers::Openstack::StorageManager::CinderManager::CloudVolume")
-  end
+    def assert_specific_cloud_volume
+      vol = @ems.cloud_volumes.find_by(:name => "manageiq-spec-vol")
+      expect(vol.name).to eq("manageiq-spec-vol")
+      expect(vol.status).to eq("available")
+      expect(vol.volume_type).to eq("__DEFAULT__")
+      expect(vol.type).to eq("ManageIQ::Providers::Openstack::StorageManager::CinderManager::CloudVolume")
+    end
 
-  def assert_specific_host_aggregates
-    aggregate = @ems.host_aggregates.find_by(:name => "test-aggregate")
-    expect(aggregate.name).to eq("test-aggregate")
-    expect(aggregate.type).to eq("ManageIQ::Providers::Openstack::CloudManager::HostAggregate")
-  end
+    def assert_specific_host_aggregates
+      aggregate = @ems.host_aggregates.find_by(:name => "manageiq-spec-aggregate")
+      expect(aggregate.name).to eq("manageiq-spec-aggregate")
+      expect(aggregate.type).to eq("ManageIQ::Providers::Openstack::CloudManager::HostAggregate")
+    end
 
-  def assert_specific_cloud_network
-    cn = @ems.cloud_networks.find_by(:name => "test-network")
-    expect(cn.name).to eq("test-network")
-    expect(cn.type).to eq("ManageIQ::Providers::Openstack::NetworkManager::CloudNetwork::Private")
-  end
+    def assert_specific_cloud_network
+      cn = @ems.cloud_networks.find_by(:name => "manageiq-spec-network")
+      expect(cn.name).to eq("manageiq-spec-network")
+      expect(cn.type).to eq("ManageIQ::Providers::Openstack::NetworkManager::CloudNetwork::Private")
+    end
 
-  def assert_specific_cloud_subnet
-    cs = @ems.cloud_subnets.find_by(:name => "test-subnet")
-    expect(cs.name).to eq("test-subnet")
-    expect(cs.cidr).to eq("192.168.222.0/24")
-    expect(cs.network_protocol).to eq("ipv4")
-    expect(cs.type).to eq("ManageIQ::Providers::Openstack::NetworkManager::CloudSubnet")
-  end
+    def assert_specific_cloud_subnet
+      cn = @ems.cloud_networks.find_by(:name => "manageiq-spec-network")
+      cs = @ems.cloud_subnets.find_by(:cloud_network_id => cn.id)
+      expect(cs.cidr).to eq("10.0.0.0/21")
+      expect(cs.network_protocol).to eq("ipv4")
+      expect(cs.type).to eq("ManageIQ::Providers::Openstack::NetworkManager::CloudSubnet")
+    end
```

Comment:
We were actually asserting the wrong subnet here before (subnet created in rake task doesn't have a name)